state/types.go

```diff
@@ -153,28 +156,28 @@ type StoreTxEGPData struct {
 
 // ZKCounters counters for the tx
 type ZKCounters struct {
-	GasUsed              uint64
-	UsedKeccakHashes     uint32
-	UsedPoseidonHashes   uint32
-	UsedPoseidonPaddings uint32
-	UsedMemAligns        uint32
-	UsedArithmetics      uint32
-	UsedBinaries         uint32
-	UsedSteps            uint32
-	UsedSha256Hashes_V2  uint32
+	GasUsed          uint64
```

Comment:
Rename GasUsed to Gas, since we have removed the word "used" from the other counters